engine/tests/story/test_choice_availability.py

```
@@ -147,6 +156,16 @@ def test_failing_predicate_marks_choice_unavailable(self) -> None:
 
         assert fragments and fragments[0].available is False
 
+    def test_failing_action_predicate_marks_choice_unavailable(self) -> None:
+        graph, start, _end, action = _graph_with_choice()
```

Ruff (0.15.12)

[warning] Unpacked variable `graph` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)